Library/Homebrew/diagnostic.rb

```diff
@@ -953,67 +702,6 @@ def check_for_bad_python_symlink
         EOS
       end
 
-      def check_for_non_prefixed_coreutils
-        coreutils = Formula["coreutils"]
-        return unless coreutils.any_version_installed?
-
-        gnubin = %W[#{coreutils.opt_libexec}/gnubin #{coreutils.libexec}/gnubin]
-        return if (paths & gnubin).empty?
-
-        <<~EOS
-          Putting non-prefixed coreutils in your path can cause gmp builds to fail.
-        EOS
-      rescue FormulaUnavailableError
-        nil
-      end
-
-      def check_for_non_prefixed_findutils
```

Comment:
I think things are still likely to try to pick these up if they're in /usr/local/bin regardless of env filtering